pyccel/codegen/printing/cwrappercode.py

```diff
@@ -139,7 +148,7 @@ def _print_PyArg_ParseTupleNode(self, expr):
         pyarg = expr.pyarg
         pykwarg = expr.pykwarg
         flags = expr.flags
-        args = ', '.join(['{}'.format(self._print(a)) if a.is_pointer else '&{}'.format(self._print(a)) for a in expr.args])
+        args = ', '.join(['{}'.format(self._print(VariableAddress(a))) for a in expr.args])
```

Comment:
PyArg_ParseTupleNode shoudl always get the address of the variable (pointer or not)

Reply:
Yes sorry I must have missed this one

Reply:
Ah sorry I see what you mean